modules/ROOT/pages/create-custom-rulesets.adoc

```diff
@@ -2,115 +2,31 @@
 
-To create a custom ruleset, use one of the following approaches:
-
-* <<custom-ruleset-modify>>
-* <<custom-ruleset-new>>
-
-[[custom-ruleset-modify]]
-== Create Custom Rulesets by Modifying Provided Rulesets
-To create custom governance rulesets based on rulesets that are already in Exchange, download and modify the rulesets and then upload them as new assets. This is the best approach if a provided ruleset meets most of your needs and you need to make only a few changes. 
-
-To create a custom ruleset based on one already in Exchange, use one of the following approaches:
-
-* <<use-clone-cli>>
-
-* <<manual-edit>>
-
-[[use-clone-cli]]
-=== Change Rule Severity or Enable or Disable Rules Using the CLI
-
-To create a custom ruleset using the CLI, use the `governance:ruleset:clone` command.
-
-include::anypoint-cli::partial$api-governance.adoc[tag=governance-ruleset-clone,leveloffset=+2]
-
-To list the rules in a ruleset, use the `governance:ruleset:info` command.
-
-include::anypoint-cli::partial$api-governance.adoc[tag=governance-ruleset-info,leveloffset=+2]
-
-include::3.x@anypoint-cli::partial$api-governance.adoc[tag=exchange-asset-identifier,leveloffset=+2]
-
-[[manual-edit]]
-=== Manually Modify Rulesets
-
-To create a custom ruleset manually:
-
-. Download the ruleset from Exchange.
-. Modify the ruleset YAML file. For example, update severity values or delete rules you do not want to use.
-. Validate the changes using the `governance:ruleset:validate` command. See <<validate-ruleset>>.
-. Generate the ruleset file as an asset document. See <<generate-ruleset-doc>>.
-. Publish your custom ruleset as a new asset using the Exchange UI or the `anypoint-cli exchange asset upload` command. See <<publish-to-exchange>>.
-
-[[custom-ruleset-new]]
-== Create Completely New Custom Rulesets
+If you need a ruleset other than those provided, you can create your own custom ruleset. 
 
-If you want a custom ruleset that cannot be created by modifying one of the provided rulesets:
+== Before You Begin 
 
-. Search the https://help.mulesoft.com/s/ideas[MuleSoft Ideas Portal^] for ideas for new rulesets that have already been submitted. If you do not find your idea there, consider submitting your idea in the MuleSoft Ideas Portal.
-+ 
-. To scaffold a ruleset from a data schema using CLI commands, see <<scaffold-rulesets>>.
-+
-. To learn more about how rulesets are written and write your own:
-** See the https://a.ml/docs/amf/using-amf/amf_custom_validation[AMF Custom Validation section^] in the AML Open Source project. 
-** See the https://github.com/aml-org/amf-custom-validator/blob/develop/docs/validation_tutorial/validation.md[AMF Rulesets tutorial] to learn how to author and test new rules.
+Creating custom rulesets requires installing and running CLI commands and depending on the approach you take, writing modeling language code. If you aren't familiar with CLI and modeling languages, consider enlisting help creating your custom rulesets. 
```

Comment:
Would replace "modeling language" code by "ruleset language"

Reply:
> If you aren't familiar with CLI and modeling languages, consider enlisting help creating your custom rulesets

Wouldn't say this. Sets expectations for the experience being difficult and requiring help (it's true but wouldn't include it in the doc).

Reply:
Thanks, @Nicolas Schejtman. The intention wasn't to discourage people from writing rulesets, but to set the audience expectation for this section of the doc. We know from the past year plus that some customers are not being successful with even running the basic CLI from examples. Governance officers probably aren't typically the right ones to be writing rulesets, and to be successful writing their own, customers should get the right resources to write them. 
As a developer, what language would you use to convey that one should have a certain level of technical skill to attempt this? It is difficult for someone who doesn't have the skills, and we want to set them up for success.

Reply:
How about something like the following:
By writing your own rules, you can more effectively govern your APIs according to your organization's specific regulations and standards. Take some time to get familiar with the CLI commands and ruleset language concepts before you start customizing rulesets.